modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,182 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="replacing-a-bare-metal-control-plane-node_{context}"]
+= Replacing a bare-metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} control plane node.
+
+[IMPORTANT]
+====
+If you reuse the `BareMetalHost` objects from an existing control plane host, do not leave the `externallyProvisioned` field set to `true`.
+
+Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by {product-title}.
+====
+
+.Procedure
+
+. Remove the old `baremetalhost`, machine and node objects:
++
+[source,terminal]
+----
+$ oc delete node kni1-vmaster-0
+$ oc delete bmh -n openshift-machine-api vmaster-0
+$ oc delete machine -n openshift-machine-api kni1-master-0
+----
++
+. Create the new `baremetalhost` object and the secret to store the BMC credentials:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ocp-master-2-bmc-secret
+  namespace: openshift-machine-api
+data:
+  password: <username>
+  username: <password>
+type: Opaque
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: ocp-master-2
+  namespace: openshift-machine-api
+spec:
+  automatedCleaningMode: disabled
+  bmc:
+    address: redfish-virtualmedia+http://192.168.124.113:8000/redfish/v1/Systems/f87eaf82-b32d-4291-ace6-b28677964e78
+    credentialsName: ocp-master-2-bmc-secret
+  bootMACAddress: aa:aa:aa:aa:ab:03
+  bootMode: UEFI
+  externallyProvisioned: false
+  hardwareProfile: unknown
+  online: true
+EOF
+----
++
+After the inspection is complete, the `baremetalhost` object is created and available to be provisioned.
+
+. View available `baremetalhost` objects:
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
```

Comment:
The BMC urls don't match... the ipmi ones were from an example I did back in the day, now I've switched to redfish... maybe you can use:

```
NAME           STATE                    CONSUMER                   ONLINE   ERROR   AGE
kni1-master-0  available                ocp-hkw9p-master-0         true             1h10m
kni1-master-1  externally provisioned   ocp-hkw9p-master-1         true             4h53m
kni1-master-2  externally provisioned   ocp-hkw9p-master-2         true             4h53m
kni1-worker-0  provisioned              ocp-hkw9p-worker-0-ktmmx   true             4h53m
kni1-worker-1  provisioned              ocp-hkw9p-worker-0-l2zmb   true             4h53m

Reply:
Updated as suggested.